crates/ads126x/src/lib.rs

```diff
@@ -41,12 +44,47 @@ pub enum ADCCommand {
     WREG(Register, u8), // (register address, number of registers)
 }
 
-impl<SPI> ADS126x<SPI>
+fn delay(delay: u32) {
```

Comment:
This is added since normally one would call a Delay object from the HAL that is clocked by SysTick to delay when needed, but RTIC consumes SysTick. Not feasible (IMO) to propagate RTIC delays down to this level. Possible that an implementation of Delay could be created with a clock object instead of SysTick but I haven't look into this in depth.

Reply:
Is delay in μs, and is `nop` guaranteed to take precisely 1 clock cycle? If `nop` takes 1 clock cycle, then we'd need to go 0..(delay * cycles/delay_unit_of_time) no?